platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEventTest.java

```diff
@@ -0,0 +1,154 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Parcel;
+import android.support.test.runner.AndroidJUnit4;
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.mapbox.android.telemetry.Event;
+import com.mapbox.mapboxsdk.Mapbox;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class PerformanceEventTest {
+
+
+  @Test
+  public void checksPerformanceEventWithMetaData() throws Exception {
+
+    Event event = obtainPerformanceEvent();
+    assertTrue(event instanceof PerformanceEvent);
+
+    PerformanceEvent performanceEvent = (PerformanceEvent) event;
```

Comment:
instead of instanceof + casting, why not change return type of obtain performance event to PerformanceEvent?